x-pack/plugins/security_solution/server/fleet_integration/fleet_integration.test.ts

```diff
@@ -156,198 +171,246 @@ describe('ingest_integration tests ', () => {
               clusterName,
               isServerlessEnabled
             )
-          ),
-        },
-        artifact_manifest: { value: manifest },
-      },
-    });
+          );
 
-    const invokeCallback = async (manifestManager: ManifestManager): Promise<NewPackagePolicy> => {
-      const callback = getPackagePolicyCreateCallback(
-        logger,
-        manifestManager,
-        requestContextFactoryMock.create(),
-        endpointAppContextStartContract.alerting,
-        licenseService,
-        exceptionListClient,
-        cloudService,
-        productFeaturesService
-      );
+          if (!cloudConfig.isCloudEnabled) {
+            return policy;
+          }
 
-      return callback(
-        createNewPackagePolicyMock(),
-        soClient,
-        esClient,
-        requestContextMock.convertContext(ctx),
-        req
-      );
-    };
+          set(policy, 'windows.antivirus_registration.mode', AntivirusRegistrationModes.sync);
```

Comment:
why do we need to set this for cloud? it should be `sync` for Endpoint as well, so there should be no difference i think